util/otbn_build.py

```diff
@@ -184,34 +224,42 @@ def main() -> int:
         out_elf = out_dir / (app_name + '.elf')
         call_otbn_ld(obj_files, out_elf, linker_script = args.linker_script)
 
-        # Use objcopy to create an ELF that can be linked into a RISC-V binary
-        # (to run on Ibex). This should set flags for all sections to look like
-        # rodata (since they're not executable on Ibex, nor does it make sense
-        # for Ibex code to manipulate OTBN data sections "in place"). We name
-        # them with a .otbn prefix, so end up with e.g. .rodata.otbn.text and
-        # .rodata.otbn.data.
+        # out_elf is a fully-linked OTBN binary, but we want to be able to use
+        # it from Ibex, the host processor. To make this work, we generate an
+        # ELF file that can be linked into the Ibex image.
+        #
+        # This ELF contains all initialised data (the .text and .data
+        # sections). We change the flags to treat them like rodata (since
+        # they're not executable on Ibex, nor does it make sense for Ibex code
+        # to manipulate OTBN data sections "in place") and add a .rodata.otbn
+        # prefix to the section names.
+        #
+        # The symbols exposed by the binary will be relocated as part of the
+        # link, so they'll point into the Ibex address space. To allow linking
+        # against multiple OTBN applications, we give the symbols an
+        # application-specific prefix. (Note: This prefix is used in driver
+        # code: so needs to be kept in sync with that).
         #
-        # Symbols that are exposed by the binary (including those giving the
-        # start and end of imem and dmem) will be relocated as part of the
-        # link, so they'll give addresses in the Ibex address space. So that
-        # the RISC-V binary can link multiple OTBN applications, we give them
-        # an application-specific prefix. (Note: This prefix is used in
-        # sw/device/lib/runtime/otbn.h: so needs to be kept in sync with that).
-        sym_pfx = '_otbn_app_{}_'.format(app_name)
+        # As well as the initialised data and relocated symbols, we also want
+        # to add (absolute) symbols that have the OTBN addresses of the symbols
+        # in question. Unfortunately, objcopy doesn't seem to have a "make all
+        # symbols absolute" command, so we have to do it by hand. This also
+        # means constructing an enormous objcopy command line :-/ If we run out
+        # of space, we might have to use elftools to inject the addresses after
+        # the objcopy.
+        host_side_pfx = '_otbn_local_app_{}_'.format(app_name)
```

Comment:
`pfx` -> `prefix`. Let's not use obscure abbreviations for their own sake.

Reply:
If you feel that "pfx" is an unusual abbreviation of "prefix", maybe you could post a PR expanding it? Personally, I would have thought that it's pretty standard (like "obj" for "object" on the next line!)